### ISSUE TITLE
How to use BatchNorm2dToQuantScaleBias

### ISSUE THREAD

**Issue by MiguelAReis** [Xilinx/brevitas#393]
Hello, Im trying to quantize the BatchNormd2d Layer but when im using BatchNorm2dToQuantScaleBias the loss isn't converging(stays at the same value). When using regular nn.BatchNorm2d with the rest of the layers quantized training is possible so the problem is in the quantized BatchNorm. I couldnt find any examples on how to use this type of layers.
Thanks

**Comment by Rellek72** [comment]
Bumping this, would really appreciate a simple example beyond the basic functionality validation of the `test_merge_bn.py` file.
